Bottom: 92ad0297bd3a63194218ad2fdd9f1192c9a3dda6
Top:    5f4b653e5c2c822bc2547daef83bf9e0d463868c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 23:15:19 +0000

Temp


---

diff --git a/src/polygon.c b/src/polygon.c
index 7a47ae8..e9cac40 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2385,6 +2385,8 @@ POLYAREA *board_outline_poly (bool include_holes)
   GList *pieces_to_delete = NULL;
   bool any_pieces_kept = false;
 
+  return NULL;
+
 #define BLOAT_WORLD MIL_TO_COORD (10)
 
   whole_world = RectPoly (-BLOAT_WORLD, BLOAT_WORLD + PCB->MaxWidth,
